src/messagequeue/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+# Create your models here.
+
+class MessageTypes:
+    PREF_CHANGE = 'PC'
+
+    CHOICES = [
+        (PREF_CHANGE, 'Preference Change'),
+    ]    
+
+class Message(models.Model):
+    type = models.CharField(max_length=10, choices=MessageTypes.CHOICES)
+    timestamp = models.DateField()
+    content = models.TextField(max_length=500)
```

Comment:
Is this a permanent new thing or a temporary thing for testing?

Reply:
This might be unnecessary unless we want to be able to hold messages in the database for asynchronous processing. My thoughts were going somewhere like this (though the implementation is not finished):

- create a Message model
- use the Signals to have relevant areas of code listen for the creation of Message objects relevant to them (via the `type` field)
- have the relevant area of code responsible for deleting a message once it has been properly acted on